test/e2e/redundancy/gateway_failover.go

```diff
@@ -78,16 +78,7 @@ func testGatewayPodRestartScenario(f *subFramework.Framework) {
 	gatewayNodes := framework.FindGatewayNodes(primaryCluster)
 	Expect(gatewayNodes).To(HaveLen(1), fmt.Sprintf("Expected only one gateway node on %q", primaryClusterName))
 	framework.By(fmt.Sprintf("Found gateway on node %q on %q", gatewayNodes[0].Name, primaryClusterName))
-
-	submEndpoint := f.AwaitSubmarinerEndpoint(primaryCluster, subFramework.NoopCheckEndpoint)
-
 	// The cable driver name can't be imported from pkg/cable/wireguard to avoid breaking the subctl build on Windows
```

Comment:
My previous comment wasn’t clear — I meant this comment should be preserved along with the use of `"wireguard"` in the new code :wink:. So drop it from `gateway_failover.go` and add it to `endpoint_handler.go`.

Reply:
Not sure I understand your point, do you mean if wierguard is imported from pkg/routeagent_driver/handlers/kubeproxy/endpoint_handler.go it will break  subctl build for Windows?

I can see that Wireguard already imported from pkg/routeagent_driver/handlers/kubeproxy/route_iface.go

Reply:
Ah, that’s correct, this change only affects the route agent and doesn’t end up in `subctl`, so the comment is irrelevant and should just be dropped entirely.